### PR TITLE
Move campus id on user to a person attribute value

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-api/src/data/auth/index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { Auth } from '@apollosproject/data-connector-rock';
 const { schema, resolver, contextMiddleware } = Auth;
 
 class dataSource extends Auth.dataSource {
+  expanded = true;
```
Comment (conrad-vanl) [inline packages/apollos-api/src/data/auth/index.js]: could expanding Auth have unintended performance consequences?

Comment (vinnyjth) [reply packages/apollos-api/src/data/auth/index.js]: @conrad-vanl Potentially, yes. It's something I thought about when adding that attribute but haven't got metrics on yet. The alternative is to fetch a user's attributes within the `Campus` datasource itself, which will add another request but potentially reduce the time to resolve `currentUser`. 

The good news is that fetching the campus is faster overall (since we don't have to go through the `family` attribute). We also pretty much always get both the current user and their campus in willow, so if `expanded = true` is measurably slower, there are only a few places where we will be fetching that data needlessly. 

The bad news is that in those cases where we get the current user and don't want their campus (or get a user login, the only other call in `auth` that will be expanded) we potentially will pay that price. 

I would be happy to come up with a test to validate all of these assumptions, but after thinking it through last night I'm not particularly worried. I was thinking on my ride in about making `expanded` something you could define per request in core, rather than per dataSource, which might be the best long term solution.

Comment (vinnyjth) [reply packages/apollos-api/src/data/auth/index.js]: image

Looks like `without loadAttributes` is twice as fast in my tests. 
image

**Comment by vinnyjth** [comment]
@conrad-vanl Updated to pull from attribute value table directly, seems to be the most efficient way to get at this data.

File: packages/apollos-api/src/data/campus/index.js [hunk 2]
```diff
@@ -18,6 +20,49 @@ class dataSource extends originalDataSource {
...
+
+    await this.post(
+      `/People/AttributeValue/${currentUser.id}?attributeKey=AppCampus&attributeValue=${campus.guid}`
+    );
+
+    return currentUser;
+  };
+
+  getForPerson = async ({ personId }) => {
+    const APP_CAMPUS_ATTRIBUTE_ID = 132803;
```
Comment (conrad-vanl) [inline packages/apollos-api/src/data/campus/index.js]: do you see this as a temporary hack or what you want to do long term

Comment (vinnyjth) [reply packages/apollos-api/src/data/campus/index.js]: It's what I want to do long term for Willow, assuming they don't change their mind on campus ids.
